src/util/strconv.h

```diff
@@ -106,11 +110,97 @@ static char *rm_strndup_unescape(const char *s, size_t len) {
   return ret;
 }
 
+// transform utf8 string to lowercase using nunicode library
+// encoded: the utf8 string to transform
+// in_len: the length of the utf8 string
+// dst: the destination buffer to store the transformed string
+// returns the bytes written to encoded, or 0 if the length of the transformed
+// string is greater than in_len and no transformation was done
+static size_t nunicode_tolower(const char *encoded, size_t in_len, char *dst) {
```

Comment:
Nitpick: Consider renaming the function to not mention the specific library name, e.g., `unicode_tolower`

Reply:
OK. Done in [b71f865](https://github.com/RediSearch/RediSearch/pull/5391/commits/b71f8656fa443d46a814923f21c05adf009a0ecb)

Reply:
OK. Done.